### PR TITLE
add all redirects from old site

### PR DESCRIPTION
**Pull request by spencerhunter**

### CODE REVIEW

**Comment by spencerhunter** [comment]
Just realized, will the following need to be added?
```
/guides/    /guides
/guides/send-money/    /guides/send-money
/guides/receive-money/    /guides/receive-money
/guides/transfer-money-between-users/    /guides/transfer-money-between-users
/guides/webhooks/    /guides/webhooks
/guides/auth/    /guides/auth
```

**Comment by sddaniels** [comment]
> Just realized, will the following need to be added?

It appears to automatically redirect to the canonical URL in this case. So it should be good

File: public/_redirects [hunk 1]
```diff
@@ -1,2 +1,60 @@
...
+/resources/dwolla-js/on-demand-bank-transfers.html    /guides/dwolla-js/instant-account-verification
+/resources/dwolla-plaid-integration.html    /guides/plaid
+/resources/testing.html    /guides/sandbox
+/resources/    /concepts
+/resources/account-types.html    /concepts/customer-types
+/resources/funding-source-verification.html    /guides/dwolla-js/instant-account-verification
+/resources/balance-funding-source.html    /concepts/balance-funding-source
+/resources/bank-transfer-workflow/processing-times.html    /concepts/transfer-processing-times
+/resources/same-day-ach.html    /concepts/same-day-ach
+https://developers.dwolla.com/resources/bank-transfer-workflow/transfer-failures.html    /concepts/transfer-failures
```
Comment (sddaniels) [inline public/_redirects]: Is this a typo with the full URL?

Comment (spencerhunter) [reply public/_redirects]: AAhhh, yep, sure is! good catch. I'll update

**Review by sddaniels — APPROVED** [review]

**Pull request merged**